Dockerfile.bundle

```diff
@@ -0,0 +1,11 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=opendatahub-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=beta
+LABEL operators.operatorframework.io.bundle.channel.default.v1=0.9
+
+COPY bundle/manifests /manifests/
+COPY bundle/metadata /metadata/
```

Comment:
missing newline?

Reply:
Not anymore....can't believe I did that...I hate missing newlines.